articles/connections/social/devkeys.md

```diff
@@ -53,3 +53,5 @@ If you are using the [Classic Universal Login experience](/universal-login/class
 5. `prompt=none` won't work on the [/authorize](/api/authentication/reference#social) endpoint. [Auth0.js' checkSession() method](/libraries/auth0js#using-checksession-to-acquire-new-tokens) uses `prompt=none` internally, so that won't work either.
 
 6. If Auth0 is acting as a <dfn data-key="security-assertion-markup-language">SAML</dfn> Identity Provider and you use a social connection with the Auth0 developer keys, the generated SAML response will have some errors, like a missing `InResponseTo` attribute or an empty `AudienceRestriction` element.
+
+7. [Multi-factor Authentication in Auth0](/mfa) will not function properly. This is because when the MFA authentication is successful will do a post in `https://${account.namespace}/mf`. When using Auth0's developer keys, the session is established on a special endpoint that is generic and tenant agnostic, and calling `/mf` will not find your previous session, resulting in an error
```

Comment:
Awkward syntax: This is because when the MFA authentication is successful will do a post in...
When MFA authentication is successful, a post will generate in...

Reply:
Looks better now?

Reply:
Yes, thank-you!